### PR TITLE
UI home page setup g1

### LINKED ISSUE #18: Integrate Bootstrap with Angular
Integrate boostrap with angular and record the process

### LINKED ISSUE #19: Set up initial home page/landing page
Design and setup the initial landing/homepage of the site - mockup available in google drive

### PR DESCRIPTION
**Pull request by g1isgone**
* Initial UI home page with animation and navbar
* Bootstrap installation + Animation installation 
* README update for npm installation

### CODE REVIEW

**Review by elizabethcarney — COMMENTED** [review]
The home page looks so awesome!! The animation is super smooth and modern-looking. A few very small comments:
- I might make the nav menu font size bigger so that it's more eye-catching, especially since our site has multiple pages instead of a scrolling layout -- but interested in what other people think
- Is there a slight delay before the animation starts? Can't tell if it's just my internet or if it's built-in
- Can we make the dropdown resources menu responsive to the page width? Right now I have to scroll to the right for it to be fully in view. Either the text could wrap to the next line if it doesn't fit, or the right side of the menu could align with the right side of the window

I'm so excited to see the website taking shape!!

File: README.md [hunk 1]
```diff
@@ -1 +1,15 @@
-Smith Tech Page - Link to Kanban Git
+Smith Tech Page 
+* We use bootstrap for this website, before starting you may have to run the following commands in the scs-app/ folder: 
+
+1) npm install popper.js --save 
+2) npm install query --save
+3) npm install tether --save 
+4) npm install bootstrap --save 
+5) npm install bootstrap-icons --save
+
```
Comment (elizabethcarney) [inline README.md:9]: Suggestion: add 
6) npm install jquery --save

Comment (g1isgone) [reply README.md:9]: ah good one - I forgot to add that will re-raise pr or put in separate commit

**Comment by g1isgone** [comment]
> The home page looks so awesome!! The animation is super smooth and modern-looking. A few very small comments:
> 
> * I might make the nav menu font size bigger so that it's more eye-catching, especially since our site has multiple pages instead of a scrolling layout -- but interested in what other people think
**Good point :)**
> * Is there a slight delay before the animation starts? Can't tell if it's just my internet or if it's built-in 
**Yes I added a delay animation - if you / other people don't like it I can take it out  : )**
> * Can we make the dropdown resources menu responsive to the page width? Right now I have to scroll to the right for it to be fully in view. Either the text could wrap to the next line if it doesn't fit, or the right side of the menu could align with the right side of the window
**oh hmmm I see let me check - maybe I need to restrict the size / length of the dropdown box or I could also change the location of events and resourcesso it's in full view**
> 
> I'm so excited to see the website taking shape!!

File: README.md (hunk 1 above)
Comment (g1isgone) [reply README.md:9]: realized the query was a typ-o lol

Comment (elizabethcarney) [reply README.md:9]: Ohh my gosh i should've realized that lol sorry

Comment (g1isgone) [reply README.md:9]: haha np was my typo anyway lol

**Pull request merged**